battles/helpers.py

```diff
@@ -75,22 +72,21 @@ def order_battle_pokemons(cleaned_data):
 def send_battle_result_email(battle):
     battle.refresh_from_db()
     trainers = [battle.trainer_creator, battle.trainer_opponent]
+    trainer_creator_team = TrainerTeam.objects.get(
+        battle_related=battle, trainer=battle.trainer_creator)
+    trainer_opponent_team = TrainerTeam.objects.get(
+        battle_related=battle, trainer=battle.trainer_opponent)
+
     for trainer in trainers:
```

Comment:
Do you still need to iterate in a `for` loop here, making a call to `send_templated_email` for each trainer, or could you call it just once, adding both trainers to the `bcc` list or something like that? I might be overlooking this, but it seems to me like the email contents are exactly the same.

Reply:
Done!